lib/media_query_wrapper.dart

```diff
@@ -1,26 +0,0 @@
-import 'dart:ui';
-
-import 'package:flutter/material.dart';
-import 'package:flutter_riverpod/flutter_riverpod.dart';
-import 'package:sweyer/sweyer.dart';
-
-class MediaQueryWrapper extends ConsumerWidget {
-  const MediaQueryWrapper({
-    super.key,
-    required this.child,
-  });
-
-  final Widget child;
-
-  @override
-  Widget build(BuildContext context, WidgetRef ref) {
-    final textScaleFactor = ref.watch(textScaleFactorStateNotifierProvider.select((value) => value));
-
-    return MediaQuery(
-      data: MediaQueryData.fromWindow(WidgetsBinding.instance.window).copyWith(
```

Comment:
The `MediaQueryWrapper` was not updating the `MediaQueryData` correctly.

In the function documentation of `MediaQueryData.fromWindow` it says:
```
If you use this, you should ensure that you also register for
notifications so that you can update your [MediaQueryData] when the
window's metrics change. For example, see
[WidgetsBindingObserver.didChangeMetrics] or
[dart:ui.PlatformDispatcher.onMetricsChanged].
```
This was fixed by using the MediaQuery provided by `MaterialApp.router`, which updated correctly.